project/src/main/webapp/script2.js

```diff
@@ -0,0 +1,8 @@
+function listPollingLocations() {
+  var address = document.getElementById("location-input").element[0].value;
+
+  fetch(`/pollingLocation?address=${address}`).then(response => response.json()).then((address) => {
+    const pollAddress = document.getElementById('poll-address');
+    pollAddress.appendChild(address);
```

Comment:
Maybe use a different name than "address", which you already used earlier to reference a different thing